autogpts/autogpt/tests/integration/agent_factory.py

```diff
@@ -1,22 +1,9 @@
 import pytest
+from forge.config.ai_profile import AIProfile
+from forge.config.config import Config
+from forge.file_storage import FileStorageBackendName, get_storage
 
 from autogpt.agents.agent import Agent, AgentConfiguration, AgentSettings
-from autogpt.agents.prompt_strategies.one_shot import OneShotAgentPromptStrategy
-from autogpt.config import AIProfile, Config
-from autogpt.file_storage import FileStorageBackendName, get_storage
-from autogpt.memory.vector import get_memory
-
-
-@pytest.fixture
-def memory_json_file(config: Config):
-    was_memory_backend = config.memory_backend
-
-    config.memory_backend = "json_file"
-    memory = get_memory(config)
-    memory.clear()
-    yield memory
-
-    config.memory_backend = was_memory_backend
```

Comment:
was this just not used?

Reply:
Yes, all memory is dead code. Memory handling should be a component.